flow/Platform.actor.cpp

```diff
@@ -680,7 +680,7 @@ const char* getInterfaceName(const IPAddress& _ip) {
 #if defined(__linux__)
 void getNetworkTraffic(const IPAddress& ip, uint64_t& bytesSent, uint64_t& bytesReceived, uint64_t& outSegs,
                        uint64_t& retransSegs) {
-	INJECT_FAULT( platform_error, "getNetworkTraffic" ); // Even though this function doesn't throw errors, the equivalents for other platforms do, and since all of our simulation testing is on Linux...
+	INJECT_FAULT( platform_error, "getNetworkTraffic" ); // getNetworkTraffic: Even though this function doesn't throw errors, the equivalents for other platforms do, and since all of our simulation testing is on Linux...
```

Comment:
How does clang-format feel about this? I wonder if we need some `// clang-format off` `// clang-format on` blocks since it's so long

Reply:
I thought about this as well... For now we don't use this, but in the future I would like to change the way these coverage files are generated to something that is syntactically more robust (one option would be to have a command line option to generate the coverage file from the binary)